contributors.yaml

```diff
@@ -0,0 +1,276 @@
+    - Software development
+- given-names: Joseph
+  family-names: Mfutso-Bengo
+  orcid: "https://orcid.org/0000-0001-9396-3700"
+  affiliation: "Kamuzu University of Health Sciences"
+  website: "https://www.researchgate.net/profile/Joseph-Mfutso-Bengo"
+  role: "Lead policy translation"
+  contributions:
+    - Policy translation
+- given-names: Tim
+  family-names: Colbourn
+  orcid: "https://orcid.org/0000-0002-6917-6552"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/3084"
+  github-username: timcolbourn
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Joseph
+  family-names: Collins
+  orcid: "https://orcid.org/0000-0003-4801-7218"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/70734"
+  github-username: joehcollins
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Iwona
+  family-names: Hawryluk
+  orcid: "https://orcid.org/0000-0001-6919-4366"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/i.hawryluk19"
+  github-username: ihwaryluk
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Britta
+  family-names: Jewell
+  orcid: "https://orcid.org/0000-0003-4275-8509"  # to check
+  affiliation: "Imperial College London" # to check
+  website: https://www.imperial.ac.uk/people/b.jewell
+  github-username: brittajewell
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Ines
+  family-names: Li Lin
+  orcid: "https://orcid.org/0000-0002-1945-3208"
+  affiliation: "University College London"
+  github-username: inesll
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Tara
+  family-names: Mangal
+  orcid: "https://orcid.org/0000-0001-9222-8632"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/t.mangal"
+  github-username: tdm32
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Robbie
+  family-names: Manning-Smith
+  orcid: "https://orcid.org/0000-0002-9457-2821"
+  affiliation: "University College London"
+  github-username: RobertManningSmith
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Sakshi
+  family-names: Mohan
+  orcid: "https://orcid.org/0000-0002-1268-1769"
+  affiliation: University of York
+  website: "https://www.york.ac.uk/che/staff/research/sakshi-mohan/"
+  github-username: sakshimohan
+  contributions:
+    - Epidemiology and modelling
+    - Health economics
+    - Software development
+- given-names: Wingston
+  family-names: Ng'ambi
+  orcid: "https://orcid.org/0000-0003-3967-905X"
+  affiliation: "Kamuzu University of Health Sciences"
+  website: "https://www.linkedin.com/in/wingston-ng-ambi-7950707b"
+  github-username: winnga # to check
```

Comment:
@tbhallett Do you happen to know if this is GitHub username is Wingston's (it is listed as having some commits associated with it in the GitHub contributors list and Wingston's name seemed most plausible match).